fs/dafsserver/dafsserver.cpp

```diff
@@ -1119,10 +1119,21 @@ class CRemoteDiskBaseActivity : public CSimpleInterfaceOf<IRemoteReadActivity>,
     bool eofSeen = false;
     const RtlRecord *record = nullptr;
     RowFilter filters;
+    RtlDynRow *filterRow = nullptr;
     // virtual field values
     StringAttr logicalFilename;
     unsigned numInputFields = 0;
 
+    inline bool fieldFilterMatch(const void * buffer)
+    {
+        if (filterRow)
+        {
+            filterRow->setRow(buffer, 0);
```

Comment:
Better is 
    filterRow->setRow(buffer, filters.getNumFieldsRequired());
for rows with large numbers of variable size fields.

Reply:
ok, this is not new btw, just moved code.
Will change.